src/restrictededitingmodeediting.js

```diff
@@ -92,6 +92,13 @@ export default class RestrictedEditingModeEditing extends Plugin {
 				evt.stop();
 			}
 		}, { priority: 'highest' } );
+
+		// Set unique class to every $root element for additional styling in restricted editing
```

Comment:
```suggestion
		// Make styling restricted and standard modes differently possible.
```

Don't read the code in a comment. Just say why it's there (if that's even necessary).

Reply:
I removed this comment entirely.